app/helpers/currency_helper.rb

```diff
@@ -0,0 +1,9 @@
+module CurrencyHelper
```

Comment:
I'd personally prefer removing this class and just being explicit in the controllers but not a big deal either way, will leave it up to you.